docs/create/heroku/herokugo/index-herokugo.mdx

```diff
@@ -0,0 +1,277 @@
+---
+id: index-herokugo
+title: Deploy a Go app on Heroku using Redis
+sidebar_label: How to deploy a Go based application on Heroku using Redis 
+slug: /create/heroku/herokugo
+authors: [ajeet]
+---
+
+import RedisCard from '@site/src/theme/RedisCard';
+
+
+Heroku is a PaaS (platform as a service) for building and running software applications in the cloud. Heroku today supports programming languages such as Java, Python, Ruby, Node.js and Go. Heroku manages your app portfolio in a straightforward Dashboard or with a CLI. Heroku's horizontally scalable, share-nothing architecture is designed for building services in today's world of containerized applications. 
+
+Here are few popular terminologies used in Heroku:
+
+- Dynos: The Heroku Platform uses the container model to run and scale all Heroku apps. The containers used at Heroku are called “dynos.Dynos are isolated, virtualized Linux containers that are designed to execute code based on a user-specified command.
+- Buildpack: It is a config script for the build automation process, describing how a container image should be created.
+- Add-ons: These are tools and services for extending a Heroku application's functionality, such as data storage and processing, monitoring, or analytics.
+- Heroku CLI: It is a tool for building and running Heroku apps from within the terminal. (Docker, too, uses its own CLI for working with the platform.)
+- Git: It is a popular version control system for tracking changes to a software's source code. Heroku makes it easy to manage your app deployments with git, and has built-in integrations with the GitHub hosting platform for git repositories.
+
+Heroku recognizes an app as being written in Go by the existence of a go. mod file in the root directory. Heroku also supports govendor, godep & GB, but this tutorial focuses only on Go modules.Here's a quickstart guide to deploy Go apps on Heroku using Redis. We will be deploying a sample Chat application written in Go.
+
+### Step 1. Create a Redis Enterprise Cloud Database
+
+Create your free Redis Enterprise Cloud account by visiting [this link](https://redis.com/try-free).
+
+![recloud](try-free.png)
+
+[Follow this link to create a Redis Enterprise Cloud](/create/rediscloud) subscription and database as shown below:
+
+
+![heroku](launch_database.png)
+
+The database endpoint URL is unique so will be different in your case. Save it for future reference.
```

Comment:
They will also need the port and password later?

Reply:
Yes. Let me add it.